From 59b9ac2c1cfd23af3efe2d558c52a7faf4d3a87c Mon Sep 17 00:00:00 2001
From: Dmitry Timoshkov <dmitry@baikal.ru>
Date: Mon, 15 Sep 2025 17:22:20 +0300
Subject: [PATCH 01/17] winex11.drv: Remove a no longer valid assert().

This is a regression caused by f74900ad1a580eda8fd4923cbd4881b42b042733
that made dmDriverExtra be either to 0 or sizeof(void *).

Signed-off-by: Dmitry Timoshkov <dmitry@baikal.ru>
---
 dlls/winex11.drv/xvidmode.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/dlls/winex11.drv/xvidmode.c b/dlls/winex11.drv/xvidmode.c
index dd762f23705..f2313d88090 100644
--- a/dlls/winex11.drv/xvidmode.c
+++ b/dlls/winex11.drv/xvidmode.c
@@ -161,7 +161,6 @@ static void xf86vm_free_modes(DEVMODEW *modes)
     if (modes)
     {
         BYTE *ptr = (BYTE *)modes - sizeof(xf86vm_modes);
-        assert(modes[0].dmDriverExtra == sizeof(XF86VidModeModeInfo *));
         memcpy(&xf86vm_modes, ptr, sizeof(xf86vm_modes));
         XFree(xf86vm_modes);
         free(ptr);
-- 
2.51.2

